bitcoin/src/blockdata/witness.rs

```diff
@@ -233,6 +233,7 @@ impl Encodable for Witness {
 
 impl Witness {
     /// Creates a new empty [`Witness`].
+    #[inline]
     pub fn new() -> Self { Witness::default() }
```

Comment:
I think this could be const too if you move the code from `default` here and call `new` from `default`.

Reply:
I believe so. Looks like Vec::new has been const since 1.39.

Reply:
Nice, thanks!

Reply:
Did you mean to make this `const`?